### ISSUE TITLE
Make search box widget accessible

### ISSUE THREAD

**Issue by draperd** [draperd/CodeChallenge1#9]
The search box we create in #7 should be updated to include accessibility attributes - most likely we'll use the aria attributes. This is probably low priority to implement after getting something functionally working
